packages/jest-runtime/src/__tests__/transform-test.js

```diff
@@ -18,15 +18,10 @@ jest
 jest.mock(
   'test-preprocessor',
   () => {
-    const stableStringify = require('json-stable-stringify');
     return {
       getCacheKey: jest.fn((content, filename, configStr) => 'ab'),
       process: (content, filename, config) => {
-        return (
-          `\nScript: ${content}\n` +
-          `Path: ${filename}\n` +
-          `Config: ${stableStringify(config, {space: 2})}`
-        );
```

Comment:
can we keep this? I actually wanted to have the config and the original script in the snapshot.